src/index.js

```diff
@@ -1,25 +1,15 @@
 import Edit from './edit/index.vue';
+import info from './info';
+import Toolbar from './edit/Toolbar.vue';
 
-/**
- * A method used to create the initial state of the element by declaring the
- * defaults for the elements props. Does not need to be defined.
- */
-const initState = () => ({ });
+const initState = () => ({ fileName: null, videoId: null, playable: false });
+
+export { Edit, Toolbar };
 
-/**
- * The fields that need to be customized are:
- * name: a string that is displayed to a user in the editor
- * ui->icon: a string representing the name of the MDI (https://materialdesignicons.com/)
- * icon that is displayed to the user in the editor
- * ui->forceFullWidth: a boolean value which defines if the element can only be
- * added as full width element
- */
```

Comment:
This seems useful. Does it make sense to leave it here?

Reply:
It's not here anymore. It's in red color, so it's probably from one of the previous commits :)

Reply:
Sorry for bad wording. 😅  I've meant that it might be a good idea to revert that change and keep the comment.

Reply:
Oh sry, my bad I misunderstood that. I will revert this change 👍